src/sider/dialogmoter/motebehov/MotebehovKvitteringInnhold.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from "react";
+import { MotebehovVeilederDTO } from "@/data/motebehov/types/motebehovTypes";
+import {
+  MotebehovIkkeSvartImage,
+  MotebehovKanIkkeImage,
+  MotebehovKanImage,
+} from "../../../../img/ImageComponents";
+import { VStack } from "@navikt/ds-react";
+
+const setSvarIkon = (deltakerOnskerMote?: boolean): string => {
+  switch (deltakerOnskerMote) {
+    case true: {
+      return MotebehovKanImage;
+    }
+    case false: {
+      return MotebehovKanIkkeImage;
+    }
+    default: {
+      return MotebehovIkkeSvartImage;
+    }
+  }
+};
+
+interface MotebehovKvitteringInnholdProps {
```

Comment:
Bare kjøre `Props`? 😊
```suggestion
interface Props {
```